pom.xml

```diff
@@ -0,0 +1,703 @@
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jetbrains.kotlin</groupId>
+        <artifactId>kotlin-test</artifactId>
+        <version>${version.kotlin}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.jetbrains.kotlin</groupId>
+        <artifactId>kotlin-test-junit5</artifactId>
+        <version>${version.kotlin}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>${version.org.junit.jupiter}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>${version.org.junit.jupiter}</version>
+        <scope>test</scope>
+      </dependency>
+
+      <!--
+      Transitive Dependency Management
+
+      In this section we address build issues including security vulnerabilities
+      in transitive dependencies we don't explicitly declare above or in the submodules
+      Versioning and scope declared here will override anything inherited through
+      transitivity, so use with care. Also note: these are in place for a
+      point in time. As we maintain this software, the manual forced resolution we do
+      here may:
+
+      1) No longer be necessary (if we have removed a dependency path leading to dep)
+      2) Break an upgrade (if we upgrade a dependency and this forces a lower version
+         of a transitive dependency it brings in)
+
+      So we need to exercise care here, and, when upgrading our deps, check to see if
+      these overrides of transitive dependency versions and scope aren't breaking things.
+
+      When adding an entry here, please reference the issue which explains why we
+      needed to do this; it will help future maintainers understand if the force
+      is still valid, should be removed, or handled in another way.
+
+      When in doubt, ask! :)
+      -->
+      <!--
+
+      FasterXML Jackson -
+
+      Needed for libraries to play nicely together and avoid NoClassDefFoundError
+      -->
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-annotations</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-core</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>${version.com.fasterxml.jackson}</version>
+      </dependency>
+      <!-- Addresss https://github.com/TBD54566975/web5-kt/issues/243 -->
```

Comment:
think we can get rid of these comments, since over time they'll become stale

Reply:
@jiyoontbd syas let's leave here because, yes, still relevant as of now - and future maintainers will know if it's safe to remove and why